### ISSUE TITLE
Phippy page design v2

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#542]
Update Phippy page design as per Gareth’s new design in figma with new characters etc.

**Comment by KTan1226** [comment]
Could you please drop in a link to the Figma so I can review? @GarethAcidWorks  I'd love to see the proposed changes and get a better understanding

**Comment by cjyabraham** [comment]
[Figma file is here](https://www.figma.com/file/FARMrOYHOfrFRi9iakYRDA/CNCF_Website-Files?node-id=12%3A96). There are some open issues James has raised on the figma that need addressing. Then design needs approval before implementation can begin.

**Comment by KTan1226** [comment]
Thanks @cjyabraham - to clarify, do you need me to weigh in on the Figma comments or are they specifically for @GarethAcidWorks?

**Comment by cjyabraham** [comment]
You are welcome to weigh in on the comments and design in general, however, we also need Gareth's response to them.

**Comment by GarethAcidWorks** [comment]
Sure I have just tagged you into a comment in Figma.

You can find the page at section 17. See screenshot.

The additions we have so far is:


  1.  3d phippy scene
  2.  New characters added / really new ones also to be added
  3.  3D headshots
  4.  Video of Phippy characters turning 3d (already done

Nice to have – we’ve discussed making the profile icons a real profile so when you click on them it launches a card. This allows us to have a place for their background / story and show the link to their associated projects where appropriate.


***@***.***

From: Kristi Tan ***@***.***>
Date: Thursday, 20 October 2022 at 12:32
To: cncf/cncf.io ***@***.***>
Cc: Gareth Davies ***@***.***>, Mention ***@***.***>
Subject: Re: [cncf/cncf.io] Phippy page design v2 (Issue #542)

Could you please drop in a link to the Figma so I can review? @GarethAcidWorks<https://github.com/GarethAcidWorks> I'd love to see the proposed changes and get a better understanding

**Comment by GarethAcidWorks** [comment]
I’ve responded to these in Figma and made some additional options for the profile icons.



From: Chris Abraham ***@***.***>
Date: Thursday, 20 October 2022 at 14:09
To: cncf/cncf.io ***@***.***>
Cc: Gareth Davies ***@***.***>, Mention ***@***.***>
Subject: Re: [cncf/cncf.io] Phippy page design v2 (Issue #542)

Figma file is here<https://www.figma.com/file/FARMrOYHOfrFRi9iakYRDA/CNCF_Website-Files?node-id=12%3A96>. There are some open issues James has raised on the figma that need addressing. Then design needs approval before implementation can begin.

**Comment by thetwopct** [comment]
Adding a popup to the character heads would definitely make sense and allow for a biography which has already been requested and being tracked https://github.com/cncf/cncf.io/issues/526 
But we could add this later so as not to need content before moving forward with design changes.

**Comment by KTan1226** [comment]
Hi all! As discussed today, please share the raw Phippy video file with me and I'll get it uploaded to YT. @Charley-Mann could you please weigh in on the content side? First thing is, do we have all of the characters in the 3D style uploaded? Then, do we want to move forward with James' recommendation of the character heads/bios? 

Please let me know if there is anything I can do to help as it relates to bios/other content. Thanks!

**Comment by GarethAcidWorks** [comment]
Hi all. Here is the video. @KTan1226 i can also send it to you over slack if that helps.

As it's a fun piece of content let's write a few words to support the video on Youtube. I know @Charley-Mann is super busy right, perhaps i can mock something up and if it needed editing Charley can work her magic.

https://user-images.githubusercontent.com/91599545/201375092-ee989f49-6613-42fa-b548-8a1ace137050.mp4

**Comment by GarethAcidWorks** [comment]
Additional updates:

- [ ] new Phippy scene asset has been provided to James.
- [ ] I will create the new character profiles (Cappy etc etc)
- [ ] Store merch images below can be updates as we have some new items
- [ ] Profile pic / bio pop up approach to be confirmed

Question - @Charley-Mann  Should 'Donate a Phippy character' stand out more?

**Comment by Charley-Mann** [comment]
Hi @KTan1226 @GarethAcidWorks there's already brief bios for most of the characters in this doc: https://docs.google.com/document/d/1IrXSK9Z4D9zZiKAstdXk8DW6tZlbqrxqgetQWtnu0bw/edit?usp=sharing

Sharing which project they are related to would be nice, although we'll just have to say that Zee is Phippy's niece (this character is not related to a project).

**Comment by Charley-Mann** [comment]
> 

@GarethAcidWorks yes please, make donate a character stand out more! 

Right now, I don't have capacity to expand this program myself, but I will find a contractor to run it.  CC @KTan1226

**Comment by GarethAcidWorks** [comment]
Notes below

![phippy and friends](https://user-images.githubusercontent.com/91599545/201684995-4b158b25-f23d-4c4e-b2bd-aaaf128172e3.png)

**Comment by GarethAcidWorks** [comment]
Hi all, above is the enhanced Phippy page, i have listed the improvements below:

1. Phippy 3D scene added - notes below + in figma
2. New improved Phippy profiles added - notes below + in figma
3.  New characters added
4. Who's coming next' bio idea to add some intrigue and to link into the next section
5. Donate a Phippy section improved and moved up the page
6. Phippy video addition
7. Representation of the store merch (same / similar module) so people have a better click through experience to the store
8. Divider added at bottom of page to separate the small print copy

Notes:
1. James has flagged some technical issues which are causing some extra effort. I have removed some items to help with this but after checking in with Charley, keeping the ground / sand gradient is preferred. I have however made this shorter so it does not hit the text below. 
2. Since we don't have enough copy to have a pop up bio, the circles are preferred as they aren't referencing the Kubernetes shape and more rounded suiting the look and feel.
7. The merch section is now a link through to the store product modules so people get a merch / buying taster before they hit the store. *** They do not need to be fully working store modules and we can just replicate them a visual to give the store feel'

**Comment by thetwopct** [comment]
Once implemented and the page design is updated I will close @KTan1226

**Comment by KTan1226** [comment]
Thanks so much, @GarethAcidWorks @thetwopct!

**Comment by thetwopct** [comment]
This is currently being blocked by an asset loading issue - https://github.com/cncf/cncf.io/issues/619 - a fix is in progress.

**Comment by GarethAcidWorks** [comment]
Hi all just checking in on the status of this.

LMK if anything is needed my end. Thanks!

**Comment by KTan1226** [comment]
Hi @GarethAcidWorks - this is on hold for now until OKRs for 2023 are determined. Thanks!

**Comment by thetwopct** [comment]
The page has been updated with the revised design.
Any Editor can now easily add, remove, and update the Phippy family.

I have moved out the Shopify product embed as a separate issue (current static implementation works for now https://github.com/cncf/cncf.io/issues/713)
